cg/services/order_validation_service/errors/case_sample_errors.py

```diff
@@ -125,6 +125,11 @@ class InvalidVolumeError(CaseSampleError):
     message: str = f"Volume must be between {MINIMUM_VOLUME}-{MAXIMUM_VOLUME} μL"
 
 
+class VolumeRequiredCaseError(CaseSampleError):
```

Comment:
```suggestion
class VolumeRequiredError(CaseSampleError):
```
In general we have decided that the errors are cleaner and more readable if we do not include the "CaseSample" etc in the name.